Tests/js/models/assets/ez-contentmodel-tests.js

```diff
@@ -388,6 +417,8 @@ YUI.add('ez-contentmodel-tests', function (Y) {
         setUp: function () {
             var that = this;
 
+            Y.eZ.Version = Y.Base.create('versionModel', Y.eZ.RestModel, [], {});
```

Comment:
actually you should load the ez-versionmodel module as it's now real dependency of ez-contentmodel (same as https://github.com/ezsystems/PlatformUIBundle/pull/251/files#r33257614 actually)